CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next
+* Template updates voor Aanvragen, Checken
+* Nieuw template voor Samenwerken toegevoegd. 
```

Comment:
Een entry in de CHANGELOG per issue, vergezeld door het issue number. Zie de rest van de file. Verder worden changes gecategoriseerd. Ik denk dat dit een ADDED is:

```
## NEXT - UNRELEASED

### Added
* Documentation: "Aanvragen" en "Checken" bijgewerkt, "Samenwerken" toegevoegd (#329)
```

Reply:
Nummer staat er ook bij.